### DISCUSSION TITLE
How update Go-Whats in production mode Docker

### DISCUSSION THREAD

**Discussion by WilmarAlmeida** [aldinokemal/go-whatsapp-web-multidevice#130] [Q&A] [answered]
How update Go-Whats in production mode in Docker ? 

Can I update while keeping the connection active?

**Comment by aldinokemal** [comment]
normally any docker will have downtime while doing an update

**Comment by WilmarAlmeida** [comment]
How update ? what are the commands?

**Comment by aldinokemal** [comment, marked as answer]
docker pull aldinokemal2104/go-whatsapp-web-multidevice:latest

**Reply by WilmarAlmeida** [reply]
right... and can you tell if by updating everything it still works, or will I need to read the QrCode again?

**Reply by aldinokemal** [reply]
Everything is operational. There is no need to scan qr
